packages/js/client-config-builder/README.md

```diff
@@ -56,11 +56,16 @@ Finally, build a ClientConfig or CoreClientConfig to pass to the PolywrapClient
   // accepted by either the PolywrapClient or the PolywrapCoreClient
   let coreClientConfig = builder.build();
 
-  // build with a custom cache and/or resolver
-  coreClientConfig = builder.build(
-    new WrapperCache(),
-    RecursiveResolver.from([])
-  );
+  // build with a custom cache
+  coreClientConfig = builder.build({
+    wrapperCache: new WrapperCache(),
+  });
+
+  // or build with a custom resolver
+  coreClientConfig = builder.build({
+    resolver: RecursiveResolver.from([]),
+    wrapperCache: new WrapperCache(),
```

Comment:
It seems like you missed rebuilding the readme once here, looking at quickstart.ts 😄

Reply:
lol nice catch